README.md

```diff
@@ -2,25 +2,34 @@
 
 # Neurobagel API
     
-[![Tests](https://img.shields.io/github/actions/workflow/status/neurobagel/api/test.yaml?color=BDB76B&label=test&style=flat-square)](https://github.com/neurobagel/api/actions/workflows/test.yaml)
-[![Coverage](https://img.shields.io/coverallsCoverage/github/neurobagel/api?style=flat-square&color=8FBC8F)](https://coveralls.io/github/neurobagel/api)
-[![Python](https://img.shields.io/badge/python-3.10-4682B4?style=flat-square)](https://www.python.org/)
-[![License](https://img.shields.io/github/license/neurobagel/api?color=CD5C5C&style=flat-square)](LICENSE)
+[![Tests](https://img.shields.io/github/actions/workflow/status/neurobagel/api/test.yaml?color=BDB76B&label=test&style=flat)](https://github.com/neurobagel/api/actions/workflows/test.yaml)
+[![codecov](https://codecov.io/gh/neurobagel/api/graph/badge.svg?token=ZEOGQFFZMJ)](https://codecov.io/gh/neurobagel/api)
+[![Python](https://img.shields.io/badge/python-3.10-4682B4?style=flat)](https://www.python.org/)
+[![License](https://img.shields.io/github/license/neurobagel/api?color=CD5C5C&style=flat)](LICENSE)
 
 </div>
 
 The Neurobagel API is a REST API, developed in [Python](https://www.python.org/) using [FastAPI](https://fastapi.tiangolo.com/) and [Pydantic](https://docs.pydantic.dev/).
 
 Please refer to our [**official documentation**](https://neurobagel.org/api/) for more information on how to use the API.
 
-- [Quickstart](#quickstart)
-- [Local installation](#local-installation)
-    - [Environment variables](#set-the-environment-variables)
-    - [Docker (recommended)](#docker)
+- [Neurobagel API](#neurobagel-api)
+  - [Quickstart](#quickstart)
+  - [Local installation](#local-installation)
+    - [Clone the repo](#clone-the-repo)
+    - [Set the environment variables](#set-the-environment-variables)
+    - [Docker](#docker)
+      - [Option 1 (RECOMMENDED): Use the Neurobagel Docker Compose recipe](#option-1-recommended-use-the-neurobagel-docker-compose-recipe)
+      - [Option 2: Use the latest image from Docker Hub](#option-2-use-the-latest-image-from-docker-hub)
+      - [Option 3: Build the image using the Dockerfile](#option-3-build-the-image-using-the-dockerfile)
+      - [Send a test query to the API](#send-a-test-query-to-the-api)
     - [Python](#python)
-- [Testing](#testing)
-- [The default Neurobagel SPARQL query](#the-default-neurobagel-sparql-query)
-- [License](#license)
+      - [Install dependencies](#install-dependencies)
+      - [Launch the API](#launch-the-api)
+    - [Troubleshooting](#troubleshooting)
+  - [Testing](#testing)
+  - [The default Neurobagel SPARQL query](#the-default-neurobagel-sparql-query)
+    - [License](#license)
```

Comment:
```suggestion
  - [License](#license)
```
🍒 Hm, I know that the License section is currently a `###` subsection, but I think it makes more sense to have it on the same level as "The default Neurobagel SPARQL query". Wdyt about changing this in the README?